build/packaging/linux/bin/kcnd

```diff
@@ -74,38 +74,59 @@ start() {
         OPTIONS="$OPTIONS --multichannel"
     fi
 
+    if [[ ! -z $HTTP_ENABLE ]] && [[ $HTTP_ENABLE -eq 1 ]]; then
+        OPTIONS="$OPTIONS --http"
+        if [ ! -z $HTTP_API ]; then
+            OPTIONS="$OPTIONS --http.api $HTTP_API"
+        fi
+        if [ ! -z $HTTP_PORT ]; then
+            OPTIONS="$OPTIONS --http.port $HTTP_PORT"
+        fi
+        if [ ! -z $HTTP_ADDR ]; then
+            OPTIONS="$OPTIONS --http.addr $HTTP_ADDR"
+        fi
+        if [ ! -z $HTTP_CORSDOMAIN ]; then
+            OPTIONS="$OPTIONS --http.corsdomain $HTTP_CORSDOMAIN"
+        fi
+        if [ ! -z $HTTP_VHOSTS ]; then
+            OPTIONS="$OPTIONS --http.vhosts $HTTP_VHOSTS"
+        fi
+    fi
+
+    #Deprecated options
```

Comment:
@hyochan-brown Can I ask why you remain deprecated options?

Reply:
I heard that users don't change the config file when update the daemon.
I remain it in the old config file compatibility.

Reply:
Thanks, it helps a lot to understand!